src/main/java/com/uid2/admin/Main.java

```diff
@@ -204,15 +205,13 @@ public void run() {
             try {
                 s3KeyProvider.loadContent();
             } catch (CloudStorageException e) {
-                if (e.getMessage().contains("s3 get error")) {
+                if (e.getMessage().contains("s3 get error") || e.getMessage().contains("The specified key does not exist")) {
```

Comment:
does "s3 get error" definitely mean the file doesn't exist in s3? Worried that if we can't reach s3 for some other reason, this will overwrite the existing file.

Reply:
I don't think I know the exact answer! Probably need more input from the team

Reply:
Will make a new pr in shared to solve this